Assets/Plugins/Source/Editor/Build/EACPostBuild.cs

```diff
@@ -316,9 +316,6 @@ private static void InstallEACFiles(BuildReport report)
             string destDir = Path.GetDirectoryName(report.summary.outputPath);
             string pathToInstallFrom = GetPathToPlatformSpecificAssets(report);
 
-            if (!string.IsNullOrEmpty(pathToInstallFrom))
```

Comment:
Isn't the code as it currently is written correct? If there is no path to install the files from, how would it work?

Reply:
If it is not empty, it returns.
I guess the problem is that there is an extra `!` .